File: Java/GetTranslationsArray.java
import java.io.*;
import java.net.*;
import java.util.*;
import javax.net.ssl.HttpsURLConnection;

public class GetTranslationsArray {

// **********************************************
// *** Update or verify the following values. ***
// **********************************************

// Replace the key string value with your valid subscription key.
	static String key = "ENTER KEY HERE";

	static String host = "https://api.microsofttranslator.com";
	static String path = "/V2/Http.svc/GetTranslationsArray";

	public static String post (URL url, String content) throws Exception {
		HttpsURLConnection connection = (HttpsURLConnection) url.openConnection();
		connection.setRequestMethod("POST");
		connection.setRequestProperty("Content-Type", "text/xml");
		connection.setRequestProperty("Ocp-Apim-Subscription-Key", key);
		connection.setDoOutput(true);

        DataOutputStream wr = new DataOutputStream(connection.getOutputStream());
		byte[] encoded_content = content.getBytes("UTF-8");
		wr.write(encoded_content, 0, encoded_content.length);
		wr.flush();
		wr.close();

		StringBuilder response = new StringBuilder ();
		BufferedReader in = new BufferedReader(
		new InputStreamReader(connection.getInputStream()));
		String line;
		while ((line = in.readLine()) != null) {
			response.append(line);
		}
		in.close();

		return response.toString();
	}

	public static String GetTranslationsArray () throws Exception {
		URL url = new URL (host + path);

		String from = "en-us";
		String to = "fr-fr";

		String ns = "http://schemas.microsoft.com/2003/10/Serialization/Arrays";
		String xml =
			"<GetTranslationsArrayRequest>" +
			"  <AppId />" +
			"  <From>" + from + "</From>" +
			"  <Texts>" +
			"    <string xmlns=\"" + ns + "\">Hello</string>" +
			"    <string xmlns=\"" + ns + "\">Goodbye</string>" +
			"  </Texts>" +
			"  <To>" + to + "</To>" +
			"  <MaxTranslations>10</MaxTranslations>" +
			"</GetTranslationsArrayRequest>";
		return post (url, xml);
    }

	public static void main(String[] args) {
		try {
			String response = GetTranslationsArray ();
			System.out.println (response);
		}
		catch (Exception e) {
			System.out.println (e);
		}
	}
}

File: PHP/GetTranslationsArray.php
<?php

// NOTE: Be sure to uncomment the following line in your php.ini file.
// ;extension=php_openssl.dll

// **********************************************
// *** Update or verify the following values. ***
// **********************************************

// Replace the subscriptionKey string value with your valid subscription key.
$key = 'ENTER KEY HERE';

$host = "https://api.microsofttranslator.com";
$path = "/V2/Http.svc/GetTranslationsArray";

$params = '';

$from = "en-us";
$to = "fr-fr";

$ns = "http://schemas.microsoft.com/2003/10/Serialization/Arrays";
$content =
	"<GetTranslationsArrayRequest>" .
	"  <AppId />" .
	"  <From>$from</From>" .
	"  <Texts>" .
	"    <string xmlns=\"$ns\">Hello</string>" .
	"    <string xmlns=\"$ns\">Goodbye</string>" .
	"  </Texts>" .
	"  <To>$to</To>" .
	"  <MaxTranslations>10</MaxTranslations>" .
	"</GetTranslationsArrayRequest>";

function GetTranslationsArray ($host, $path, $key, $params, $content) {

	$headers =
		"Content-type: text/xml\r\n" .
		"Ocp-Apim-Subscription-Key: $key\r\n";

	// NOTE: Use the key 'http' even if you are making an HTTPS request. See:
	// http://php.net/manual/en/function.stream-context-create.php
	$options = array (
		'http' => array (
			'header' => $headers,
			'method' => 'POST',
			'content' => $content
		)
	);
	$context  = stream_context_create ($options);
	$result = file_get_contents ($host . $path . $params, false, $context);
	return $result;
}

$result = GetTranslationsArray ($host, $path, $key, $params, $content);

echo $result;
?>
